### PR TITLE
Update create-openapi-repo README

### PR DESCRIPTION
**Pull request by drichards1987**
Updates for the README file.

### CODE REVIEW

**Comment by sogan66** [comment]
Submit feedback approving these changes 7e4893be39a2d1ee56f44f8b7e9e68fa00a76a13

File: README.md [hunk 1]
```diff
@@ -1,141 +1,132 @@
...
+ - OpenAPI 3.0 support
+ - Split an existing OpenAPI definition into multiple files
+ - Bundle a multi-file definition into a single file
+ - Validate your OpenAPI definition using our free [openapi-cli tool](https://github.com/redocly/openapi-cli)
+ - Automate deployment of your API reference docs using CI/CD workflows
+ - Maintain code samples as separate files
  - Live editing in your editor of choice :heart_eyes:
 
-## Structure
+## Examples
```
Comment (HCloward) [inline README.md:23]: not sure about the last two examples - one is archived and the other hasn't been touched in seven years

File: README.md [hunk 2]
```diff
@@ -1,141 +1,132 @@
...
-    │   └── paths
-    │       └── README.md
-    └── package.json
-```
-
-However, you can adjust it to any structure you prefer.
+## Prerequisites
 
-The `openapi` folder is where your OpenAPI definition will live.  Inside there, and the sub-folders, there are `README.md` files to help guide you further. This is also where your entrypoint `openapi.yaml` will live.
+Before you begin, make sure you’ve installed the following prerequisites:
```
Comment (HCloward) [inline README.md:30]:
```suggestion
Before you begin, make sure you have the following prerequisites:
```

File: README.md [hunk 3]
```diff
@@ -1,141 +1,132 @@
...
-Note: the migration tool does not migrate plugins automatically. You would need to manually add them to the `transformers` folder. 
+ - `.redocly.yaml`: Configuration file for defining settings for various Redocly tools, including the lint tool and reference docs engine.
+ - `openapi`: Top-level folder that contains your OpenAPI definition, `openapi.yaml` entrypoint file, and sub-folders for `paths`, `components`, and `code_samples`.
+ - `code_samples`: Folder for organizing code samples into sub-folders, such as C# and PHP.
+ - `components`: Folder for organizing reusable components into sub-folders, such as `schema` and `response` objects.
+ - `paths`: Folder for organizing path definitions. Each path should be referenced from the `openapi.yaml` entrypoint file.
+
+## Commands
+
+The generated repository installs a dependency for our `openapi-cli` tool which supports the following commands:
```
Comment (HCloward) [inline README.md:113]: Isn't it redocly-cli now?

**Comment by lornajane** [comment]
@HCloward I suggest you edit and push to the branch directly, this pull request is many years old so apply your changes and I can always review and merge then.

(but thanks to @drichards1987 for the original PR, sorry for the slow response!)

File: README.md (hunk 3 above)
Comment (HCloward) [inline README.md:113]:
```suggestion
The generated repository installs a dependency for our `redocly-cli` tool which supports the following commands:
```

File: README.md [hunk 4]
```diff
@@ -1,141 +1,132 @@
...
+ - Validate your OpenAPI definition using our free [openapi-cli tool](https://github.com/redocly/openapi-cli)
+ - Automate deployment of your API reference docs using CI/CD workflows
+ - Maintain code samples as separate files
  - Live editing in your editor of choice :heart_eyes:
 
-## Structure
+## Examples
+- [Rebilly](https://github.com/Rebilly/RebillyAPI)
+- [Thingful](https://github.com/thingful/openapi-spec)
+- [Fitbit Plus](https://github.com/TwineHealth/TwineDeveloperDocs)
```
Comment (HCloward) [inline README.md:26]:
```suggestion
```

**Pull request merged**